tests/test_module.py

```diff
@@ -37,19 +37,19 @@ def test_passes_if_module_not_git(self, caplog):
     def test_fails_if_module_has_rackspace_http_ref(self, caplog):
         file = 'tests/test_module/module_github_has_rackspace_ref.tf'
         with Wrap(self, [file], expect_exit=True):
-            assert ('Rackspace module references should use SSH source paths FAIL in {}'.format(file)) in caplog.text
+            assert ('[FAIL] github_rackspace_module_use_ssh:Rackspace module references should use SSH source paths:some_module:{}'.format(file)) in caplog.text
 
     def test_passes_if_module_has_rackspace_ssh_ref(self, caplog):
         file = 'tests/test_module/module_git_has_rackspace_ref.tf'
         with Wrap(self, [file], expect_exit=False):
-            assert ('Rackspace module references should use SSH source paths PASS in {}'.format(file)) in caplog.text
+            assert ('github_rackspace_module_use_ssh:Rackspace module references should use SSH source paths:{}'.format(file)) in caplog.text
```

Comment:
Should include the status in this check too.

```suggestion
            assert ('[PASS] github_rackspace_module_use_ssh:Rackspace module references should use SSH source paths:{}'.format(file)) in caplog.text
```